xilem/examples/calc.rs

```diff
@@ -267,22 +276,37 @@ fn display_label(text: &str) -> impl WidgetView<Calculator> + use<> {
 /// they take up all available space in flex containers.
 fn expanded_button(
     text: impl Into<Label>,
+    is_digit: bool,
```

Comment:
I don't really hugely like the API of this component - can we either make a function which both this and `digit_button` calls, or make an enum for this field, or just inline this into `digit_button`.

The lowest lift is just to make it an enum.